src/frontend/komponenter/Behandling/Inngangsvilkår/Aleneomsorg/Bosted.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
+import {
+    Registergrunnlag,
+    SøknadOgRegistergrunnlag,
+    Søknadsgrunnlag,
+} from '../../../Felleskomponenter/Visning/DataGrunnlagIkoner';
+import { Normaltekst } from 'nav-frontend-typografi';
+import { BooleanTekst } from '../../../Felleskomponenter/Visning/StyledTekst';
+
+interface Props {
+    harSammeAdresseSøknad?: boolean;
+    harSammeAdresseRegister?: boolean;
+}
+
+const Bosted: FC<Props> = ({ harSammeAdresseSøknad, harSammeAdresseRegister }) => {
+    return (
+        <>
+            {harSammeAdresseRegister !== undefined &&
```

Comment:
Må man sjekke for både undefined og null?
Burde det ikke være tilsrekkelig å sjekke for `== null`?
Eller `!!harSammeAdresseRegister` (men kanskje er litt fult)

Trenger egentlige ikke sjekke att begge disse er null/undefined, burde være tilsrekkelig å sjekke en av de og sen sjekke om de er lik :) 

Edit: Eller bruke `harVerdi` som ble lagt til

Reply:
Enig i at det beste er å bruke `harVerdi` - men da blir TS sint, fordi nullsjekken skjer i en annen funksjon og det forstår den ikke. 

`!!harSammeAdresseRegister && !!harSammeAdresseSøknad` kan funke